[FEAT] Add iconik CA authorizer (#2)
workflow-intelligence-nexus/win.sls: README.md

    - feature_name.service.ts - It's the feature service. Its methods should implement one of the main steps of some
      feature's functionality
    - feature_name.interface.ts - This file should contain all required interfaces for the feature

> - authorizers - authorizers for iconik requests or any other requests

Reviewer: ```suggestion
- authorizers - lambda authorizers(custom authorizers) or authorizers for iconik requests
```
